apps/watch/src/components/VideoContentPage/VideoHero/VideoPlayer/VideoControls/VideoControls.tsx

```diff
@@ -344,10 +344,7 @@ export function VideoControls({
                   <AudioLanguageButton componentVariant="icon" />
                   <IconButton
                     onClick={() => setOpenSubtitle(true)}
-                    disabled={
-                      variant?.subtitle === undefined ||
-                      variant?.subtitle.length < 1
-                    }
+                    disabled={variant?.subtitleCount === 0}
```

Comment:
Sorry bro, after thinking about it can we do something like what's happening before

```suggestion
                    disabled={variant?.subtitleCount === undefined ||
                    variant?.subtitleCount < 1}
```
This just allows us to cover null values and negative and zero values